client/src/components/Tour/Tour.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div class="d-flex flex-column">
-        <UserHistories v-if="currentUser" v-slot="{ currentHistory, handlers, historiesLoading }" :user="currentUser">
+        <div v-if="currentUser">
```

Comment:
This divs condition can be attached to the parent to reduce nesting.
The condition may not even be required any longer now that the provider is dropped. I could not find any change in tour behavior when dropping it completely (without reattaching the condition)